### DISCUSSION TITLE
`HTTPS`模式下访问`redirect_url`，浏览器偶发性抛出`ERR_SSL_VERSION_OR_CIPHER_MISMATCH`

### DISCUSSION THREAD

**Discussion by despire1119** [MikeWang000000/Natter#74] [Q&A]
给大佬请晚安...
如题，在配置为`https`模式后，`Cloudflare`所代理的`redirect_url`将会抛出`ERR_SSL_VERSION_OR_CIPHER_MISMATCH，使用了不受支持的协议`。（会不会是证书问题？申请的阿里云CA证书，`direct_url`已配置成功，但`redirect_url`看样子是由`Cloudflare`代理做302跳转的，它的证书不知道应该在哪里配置?）
![ae01ddb1ddd5314ede535646c9e50b0](https://github.com/MikeWang000000/Natter/assets/10387252/ba3ad862-902b-4538-ba60-9c12aa526d25)

感谢

**Comment by MikeWang000000** [comment]
这个请自行查询。
